profiles/api/oauth2.py

```diff
@@ -147,6 +152,12 @@ def _token() -> Response:
                                      expires_at(json_data['expires_in']))
             orcid_tokens.add(orcid_token)
 
+        try:
+            orcid_record = orcid_client.get_record(profile.orcid, orcid_token.access_token)
+            profile.update_from_orcid_record(orcid_record)
+        except RequestException as exception:
+            LOGGER.exception(exception)
```

Comment:
`logger.exception` takes a message, not the exception (which already has due to global state)

Reply:
`errors.py` looks wrong too?

Reply:
yes, it is